### PR TITLE
test: reproducer for 4695

### PR DESCRIPTION
**Pull request by NGA-TRAN**
Repro for #4695 

- [x] I've read the contributing section of the project [README](https://github.com/influxdata/influxdb_iox/blob/main/README.md).
- [x] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed).

### CODE REVIEW

**Comment by NGA-TRAN** [comment]
FYI @domodwyer

File: parquet_file/src/serialise.rs [hunk 1]
```diff
@@ -123,6 +124,8 @@ where
     // Serialise the record batches into the in-memory buffer
     let meta = to_parquet(batches, meta, &mut w).await?;
 
+    debug!(?meta, "Parquet Metadata");
```
Comment (NGA-TRAN) [inline parquet_file/src/serialise.rs:127]: This will help see what statistics the `to_parquet` creates

File: parquet_file/tests/metadata.rs [hunk 2]
```diff
@@ -81,6 +81,24 @@ async fn test_decoded_iox_metadata() {
...
         "embedded metadata does not match original metadata"
     );
+
+    // Repro of 4695
+    let row_group_meta = decoded.parquet_row_group_metadata();
+    println!("ow_group_meta: {:#?}", row_group_meta);
+    assert_eq!(row_group_meta.len(), 1);
+    assert_eq!(row_group_meta[0].columns().len(), 2); // time and some_field
+    assert!(row_group_meta[0].column(0).statistics().is_some()); // There is statistics for "time"
+    assert!(row_group_meta[0].column(0).statistics().is_some()); // There is statistics for "some_field"
```
Comment (carols10cents) [inline parquet_file/tests/metadata.rs:91]: I think the second line here should be `column(1)`? Right now both these lines are testing the same thing.

```suggestion
    assert!(row_group_meta[0].column(0).statistics().is_some()); // There is statistics for "time"
    assert!(row_group_meta[0].column(1).statistics().is_some()); // There is statistics for "some_field"
```

File: parquet_file/tests/metadata.rs [hunk 3]
```diff
@@ -81,6 +81,24 @@ async fn test_decoded_iox_metadata() {
         got, meta,
         "embedded metadata does not match original metadata"
     );
+
+    // Repro of 4695
+    let row_group_meta = decoded.parquet_row_group_metadata();
+    println!("ow_group_meta: {:#?}", row_group_meta);
```
Comment (carols10cents) [inline parquet_file/tests/metadata.rs:87]: Typo, might cause debugging headaches if someone's doing ctrl-f for `row_group_meta`:

```suggestion
    println!("row_group_meta: {:#?}", row_group_meta);
```

**Review by carols10cents — CHANGES_REQUESTED** [review]
One fix and one typo :)

**Review by carols10cents — APPROVED** [review]
LGTM!

**Review by alamb — COMMENTED** [review]
👍

**Pull request merged**